airflow/utils.py

```diff
@@ -29,6 +29,8 @@
 from sqlalchemy import event, exc
 from sqlalchemy.pool import Pool
 
+import numpy
```

Comment:
It's pretty standard to do `import numpy as np`, then it'll help with respecting the 80 char per line limit we've been following to stay in line with PEP8 when possible.